SlicerProstate/SlicerProstateUtils/mixins.py

```diff
@@ -246,13 +246,13 @@ def get3DEuclideanDistance(pos1, pos2):
     return distance3D
 
   @staticmethod
-  def dilateMask(label):
+  def dilateMask(label, dilateValue, erodeValue=0):
     imagedata = label.GetImageData()
     dilateErode = vtk.vtkImageDilateErode3D()
     dilateErode.SetInputData(imagedata)
-    dilateErode.SetDilateValue(1.0)
-    dilateErode.SetErodeValue(0.0)
-    dilateErode.SetKernelSize(12, 12, 1)
+    dilateErode.SetDilateValue(dilateValue)
+    dilateErode.SetErodeValue(erodeValue)
+    dilateErode.SetKernelSize(3, 3, 3)
```

Comment:
The first Version of SliceTracker had the following:

https://github.com/SlicerProstate/SliceTracker/blob/1e48b64d8c0c334c1760ea1e9ac5fc3d700db268/SliceTracker.py#L2581-L2584